web-api/migration-terraform/main/lambdas/migrations/0035-in-care-of-to-additional-name.test.js

```diff
@@ -0,0 +1,123 @@
+const {
```

Comment:
Same for this file — I wasn’t expecting this migration (or it’s test) to be introduced here.